apps/meteor/tests/end-to-end/api/25-teams.js

```diff
@@ -1596,75 +1596,68 @@ describe('[Teams]', () => {
 		});
 
 		describe('team auto-join', () => {
-			let testTeam;
-			let createdRoom;
 			let testUser1;
 			let testUser2;
 
 			before(async () => {
-				const [testUserResult, testUser1Result] = await Promise.all([createUser(), createUser()]);
-				testUser1 = testUserResult;
-				testUser2 = testUser1Result;
-			});
-
-			beforeEach(async () => {
-				const createTeamPromise = createTeam(credentials, `test-team-name${Date.now()}`, 0);
-				const createRoomPromise = createRoom({ name: `test-room-name${Date.now()}`, type: 'c' });
-				const [testTeamCreationResult, testRoomCreationResult] = await Promise.all([createTeamPromise, createRoomPromise]);
-
-				testTeam = testTeamCreationResult;
-				createdRoom = testRoomCreationResult;
-
-				await request
-					.post(api('teams.addRooms'))
-					.set(credentials)
-					.expect(200)
-					.send({
-						rooms: [createdRoom.body.channel._id],
-						teamName: testTeam.name,
-					});
+				testUser1 = await createUser();
+				testUser2 = await createUser();
 			});
 
-			afterEach(() =>
-				Promise.all([deleteTeam(credentials, testTeam.name), deleteRoom({ roomId: createdRoom.body.channel._id, type: 'c' })]),
-			);
-
```

Comment:
this should continue in "after" if the test fails it will not remove the rooms

Reply:
oh yea, my bad